endpoints/openrtb2/auction.go

```diff
@@ -534,6 +534,12 @@ func (deps *endpointDeps) parseRequest(httpRequest *http.Request, labels *metric
 		return
 	}
 
+	// upgrade to 2.6 here
```

Comment:
nitpick: recommend renaming to "normalize to openrtb 2.6" since the incoming request may already be 2.6.